src/dsp/processor/distortion/bitcrusher.h

```diff
@@ -47,9 +50,13 @@ struct alignas(16) BitCrusher : public Processor
                         float pitch) override;
     void init_params() override;
     int tail_length() override { return tailInfinite; }
+    float note_to_pitch_ignoring_tuning(float n);
+    float dbToLinear(float n);
+    double dsamplerate_inv;
```

Comment:
Great so this works but I think we can do a wee bit better.

You'll see the template argument for BiquadFilter takes a third type which is an adapter type. We could make an adapter in a header file which implements these two tuning/db methods directly and which assumes that the object is a `SampleRateSupport` (which `Processor` is). 

Or alternately add a new interface called `BiquadSupport` which implements `note_to_pitch` and `dbToLinear` in `src/dsp/Processors.h` and then extend `SampleRateSupport` to have the double versions also - then any processor works as a bique arg if you just tag it with BiquadSupport in the inheritance list.

Reply:
Okay - I did the above. What do you think about moving the lp filter itself to BiquadSupport as well? 
Also, the build is failing in linux and mac, I'm trying to set it up in linux to debug.

Reply:
Some effects have 2 or 3 filters like a distortion with a pre and post EQ. so having distinct members makes sense - I would leave the filter as a member of the processor

Reply:
That build error is odd. I wonder if you need the full namespac3 for bitcrusher to resolve it

Reply:
That did it! Yeah it took a while for me to get it building in my Debian. 
Tested real quick again. Seems to work fine - feel free to merge if it looks good. Happy to make any other changes too!